atom/app/command_line_args.cc

```diff
@@ -43,22 +43,32 @@ const char* blacklist[] = {
   "inspect-brk",
 
   // chromium switches:
-  // find ./ -name "*switches.cc" \
-  //   | xargs grep -P --no-filename "\"\S+\";" \
-  //   | perl -pe 's|^.*?"(\S+)";|  "$1",|' \
-  //   | sort | uniq
-  "?",
+  // find ./ \( -name "*switches.cc" -o -name "*switch_list.h" \) \
+  //  | xargs grep -P --no-filename '"\S+"[);]$' \
+  //  | perl -pe 's|^.*?"(\S+)"[);]$|  "$1",|'; \
+  //  | LANG="C" sort | uniq
+  "/prefetch:1",
```

Comment:
These things don't look like the others?

Reply:
They do look different, but the slash and colon come straight from `src/content/public/common/content_switches.cc` among other switches, e.g.:

```cpp
const char kEnableSpeechDispatcher[] = "enable-speech-dispatcher";
#endif

#if defined(OS_WIN)
// /prefetch:# arguments to use when launching various process types. It has
// been observed that when file reads are consistent for 3 process launches with
// the same /prefetch:# argument, the Windows prefetcher starts issuing reads in
// batch at process launch. Because reads depend on the process type, the
// prefetcher wouldn't be able to observe consistent reads if no /prefetch:#
// arguments were used. Note that the browser process has no /prefetch:#
// argument; as such all other processes must have one in order to avoid
// polluting its profile. Note: # must always be in [1, 8]; otherwise it is
// ignored by the Windows prefetcher.
const char kPrefetchArgumentRenderer[] = "/prefetch:1";
const char kPrefetchArgumentGpu[] = "/prefetch:2";
const char kPrefetchArgumentPpapi[] = "/prefetch:3";
const char kPrefetchArgumentPpapiBroker[] = "/prefetch:4";
```